content/information-aggregation/history-stats.js

```diff
@@ -58,7 +58,7 @@ Foxtrick.modules['HistoryStats'] = {
 				var date = Foxtrick.util.time.getDateFromText(dateSpan.textContent);
 				var season = Foxtrick.util.time.gregorianToHT(date).season;
 
-				var clone = feed.querySelector('.otherEventText').cloneNode(true);
+				var clone = feed.querySelector('td + td').cloneNode(true);
```

Comment:
this selector is bound to break
```suggestion
				var clone = feed.querySelector('.float_left').cloneNode(true);
```